CalculateCalendarLogic/CalculateCalendarLogic.swift

```diff
@@ -257,6 +271,10 @@ public struct CalculateCalendarLogic {
             case (year, 4, 30, .mon) where year >= AlternateHolidaysLawYear:
                 return true
             
+            //2019年5月1日： 2019年だけ天皇の即位の日
+            case (emperorthroneYear, 5 , 1, _):
```

Comment:
__[SwiftLint]__ Case statements should vertically align with their enclosing switch statement.

<sub>posted by Sider</sub>